src/rosconsole/impl/rosconsole_glog.cpp

```diff
@@ -97,6 +97,10 @@ void register_appender(LogAppender* appender)
   rosconsole_glog_appender = appender;
 }
 
+void deregister_appender(){
+  rosconsole_glog_appender = NULL;
```

Comment:
Please use `0` here for consistency - that is the default value further up in the same file and in the other implementations.